types/evidence_test.go

```diff
@@ -46,28 +46,8 @@ func TestMaxEvidenceBytes(t *testing.T) {
 			TotalVotingPower: math.MaxInt64,
 			ValidatorPower:   math.MaxInt64,
 			Timestamp:        timestamp,
-			VoteA: makeVote(
-				t,
-				val,
-				chainID,
-				math.MaxInt32,
-				math.MaxInt64,
-				math.MaxInt32,
-				tmproto.ProposalType,
-				blockID,
-				timestamp,
-			),
-			VoteB: makeVote(
-				t,
-				val,
-				chainID,
-				math.MaxInt32,
-				math.MaxInt64,
-				math.MaxInt32,
-				tmproto.ProposalType,
-				blockID2,
-				timestamp,
-			),
+			VoteA:            makeVote(t, val, chainID, math.MaxInt32, math.MaxInt64, math.MaxInt32, 32, blockID, timestamp),
```

Comment:
Why was `32` changed to hardcoded?

Reply:
Good catch. While I fix the validation, I compared the original tendermint's test code. Then I found these differences and I try to rollback to the original tendermint's `makeVote` for avoiding the feature backports:
* https://github.com/line/ostracon/pull/513/files#diff-2d3e82f1adfa12795a56ae73d6257e30a11d9a76bae15b3a3f128b370ecc3fe0L338-R277
```
func makeVote(
	t *testing.T, val PrivValidator, chainID string, valIndex int32, height int64, round int32, step int, blockID BlockID,
	time time.Time) *Vote 
```
<img width="1201" alt="screenshot 2022-11-30 16 23 18" src="https://user-images.githubusercontent.com/1164270/204733008-ac972278-13b3-45c5-9e70-3f75bb49739a.png">

And Its usage is ambiguous, but I might be used `0x20`.
*  https://github.com/tendermint/tendermint/blob/v0.34.24/types/evidence_test.go#L36-L37
* https://github.com/tendermint/tendermint/blob/v0.34.24/types/evidence_test.go#L71
* https://github.com/tendermint/tendermint/blob/v0.34.24/types/evidence_test.go#L82-L83
* https://github.com/tendermint/tendermint/blob/v0.34.24/types/evidence_test.go#L280-L281

How about you all my thoughts? 
* No need to rollback?
* Use hex? or int?

Reply:
I think it's good to roll back to avoid feature backports. I understood why it was hardcoded after you explained it. Thanks. I resolve this conversation.